### PR TITLE
Facebook click to load test page

### PR DESCRIPTION
**Pull request by Charlie-belmer**
This adds a click to load test page which loads a large number of facebook elements, and counts the requests for display and/or download.

### CODE REVIEW

File: privacy-protections/click-to-load/index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+let facebookCalls = 0
+
+// This initializes the facebook SDK.
+window.fbAsyncInit = function() {
+  FB.init({
+      appId            : '',
+      autoLogAppEvents : true,
+      xfbml            : true,
+      version          : 'v9.0'
+    });
```
Comment (jonathanKingston) [inline privacy-protections/click-to-load/index.html:71]: @kdzwinel will tell you he's triggered by inconsistent ;'ing. (https://github.com/duckduckgo/privacy-test-pages/pull/27 will catch this in future and allow you to run `npm run test-html -- --fix` to autofix it too)

Comment (kdzwinel) [reply privacy-protections/click-to-load/index.html:71]: #27 was merged so we can get all linting issues out of the way when you rebase 🧹

File: privacy-protections/index.html [hunk 2]
```diff
@@ -16,6 +16,7 @@ <h1>Privacy Protections Tests</h1>
         <li><a href='https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking/'>Storage blocking</a></li>
         <li><a href='https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/'>Referrer trimming</a></li>
         <li><a href='http://privacy-test-pages.glitch.me/privacy-protections/https-upgrades/'>HTTPS upgrades</a></li>
+        <li><a href='http://privacy-test-pages.glitch.me/privacy-protections/click-to-load/'>Facebook click to load</a></li>
```
Comment (kdzwinel) [inline privacy-protections/index.html:19]: Can you please also add this page to the main index.html file? Sorry, it's a bit confusing that we have two 🤦‍♂️

File: privacy-protections/click-to-load/index.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+<br /><iframe src="https://www.facebook.com/plugins/video.php?href=https%3A%2F%2Fwww.facebook.com%2Ffacebook%2Fvideos%2F10153231379946729%2F&width=500&show_text=false&height=280&appId" width="500" height="280" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share" allowFullScreen="true"></iframe>
+<br /><iframe src="https://www.facebook.com/plugins/group.php?href=https%3A%2F%2Fwww.facebook.com%2Fgroups%2Fayearofrunning%2F&width=280&show_social_context=true&show_metadata=false&height=267&appId" width="280" height="267" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
```
Comment (kdzwinel) [inline privacy-protections/click-to-load/index.html:59]: This frame fails to load for me in different browsers: "Firefox Can’t Open This Page

To protect your security, www.facebook.com will not allow Firefox to display the page if another site has embedded it. To see this page, you need to open it in a new window."

**Review by kdzwinel — COMMENTED** [review]
This looks super comprehensive, thank you @Charlie-belmer ! I left couple of comments.

File: privacy-protections/index.html (hunk 2 above)
Comment (Charlie-belmer) [reply privacy-protections/index.html:19]: yeah totally forgot that one!

File: privacy-protections/click-to-load/index.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    if (resource.name.match(/facebook.com|facebook.net/)) {
+      count += 1
+    }
+  }
+  facebookCalls = count
+  document.getElementById('facebook_call_count').innerHTML = facebookCalls
+}
+
+// Calls happen over time, so periodically check calls
+setInterval(displayFacebookCalls, 500)
```
[the remark this answers is not in the document]
Comment (Charlie-belmer) [reply privacy-protections/click-to-load/index.html]: Good idea - it makes reporting a little slower, but it's a much better way of doing it.

File: privacy-protections/click-to-load/index.html (hunk 3 above)
Comment (Charlie-belmer) [reply privacy-protections/click-to-load/index.html:55]: This is true for every way I have tried including groups, but it's still in the documentation. I left it in for completeness based on the docs, but maybe I should remove it?

Comment (kdzwinel) [reply privacy-protections/click-to-load/index.html:55]: Ah, I see. Let's just leave a comment in HTML and leave it in.

**Review by kdzwinel — APPROVED** [review]
Feel free to merge after addressing that one comment 👍

File: privacy-protections/click-to-load/index.html (hunk 3 above)
Comment (Charlie-belmer) [reply privacy-protections/click-to-load/index.html:56]: I tried a few groups here, but I didn't see any different behavior. Good call on adding a comment!

**Pull request merged**